cmd/kops/toolbox_template.go

```diff
@@ -223,15 +223,45 @@ func newTemplateContext(files []string) (map[string]interface{}, error) {
 				return nil, fmt.Errorf("unable decode the configuration file: %s, error: %v", j, err)
 			}
 
-			for k, v := range ctx {
-				context[k] = v
-			}
+			context = mergeValues(context, ctx)
 		}
 	}
 
 	return context, nil
 }
 
+// Merges source and destination map, preferring values from the source map
```

Comment:
Can we make a note where we used this code from?  It is all OSS under the same language, so it is fine.  But lets give credit.